Document RootNamespace requirements in module vcxproj files

When creating a native module, the `RootNamespace` property of the project must match the namespace used in the native module. Otherwise, the project will fail to build. This is currently not documented.

Those will not match when following the getting started guide: when creating the module project `MyLibrary` is suggested as its name, whereas the sample code uses `NativeModuleSample` as its namespace.

@asklar No, it's a different error:
```
[...]\MyLibraryCPP\ReactPackageProvider.h(3,10): fatal error C1083: Cannot open include file: 'ReactPackageProvider.g.h': No such file or directory (compiling source file ReactPackageProvider.cpp)
```
The lib source I'm building [is here](https://github.com/bzoz/rnw-ui-module/tree/f79e447cd6f7a5230c0b9aa8e225fe110d80d2c8). Making [this change to the `RootNamespace`](https://github.com/bzoz/rnw-ui-module/commit/3f55c454843438f5cd9c1d34f616b90fe26a68cd) fixes the build.

I think the main issue here is that the getting started docs suggest creating `MyLibrary` as the solution name and `NativeModuleSample` as the namespace in FancyMath which doesn't work.

--- a/docs/native-modules-setup.md
+++ b/docs/native-modules-setup.md
@@ -9,7 +9,7 @@ title: Native Module Setup
 > - [Native Module Sample in `microsoft/react-native-windows-samples`](https://github.com/microsoft/react-native-windows-samples/tree/master/samples/NativeModuleSample)
 > - [Sample App in `microsoft/react-native-windows/packages/microsoft-reactnative-sampleapps`](https://github.com/microsoft/react-native-windows/tree/master/packages/sample-apps)
 
-This guide will help set you up with the Visual Studio infrastructure to author your own stand-alone native module for React Native Windows. In this document we'll be creating the scaffolding for a `MyLibrary` native module.
+This guide will help set you up with the Visual Studio infrastructure to author your own stand-alone native module for React Native Windows. In this document we'll be creating the scaffolding for a `NativeModuleSample` native module.
 
 ## Development Environment
 
@@ -44,18 +44,18 @@ The steps to create a new native module library project are:
 Follow the official React Native instructions at https://reactnative.dev/docs/native-modules-setup.
 
 ```bat
-npx create-react-native-module --module-name "MyLibrary" MyLibrary
-cd MyLibrary
+npx create-react-native-module --module-name "NativeModuleSample" NativeModuleSample
+cd NativeModuleSample
 yarn install
 ```
 
-Now you'll have a new native module project under `MyLibrary`. Be sure to look at the command output for further steps you'll want to do before publishing the project.
+Now you'll have a new native module project under `NativeModuleSample`. Be sure to look at the command output for further steps you'll want to do before publishing the project.
 
 At this point, follow the steps below to add Windows support to the newly created library.
 
 ## Adding Windows support to an existing library
 
-> The steps below are written as if you're working with the `MyLibrary` example above, in the root folder of the project. Substitute the name of the library you're actually working on where appropriate, and ensure that you're working in the appropriate root folder of the library.
+> The steps below are written as if you're working with the `NativeModuleSample` example above, in the root folder of the project. Substitute the name of the library you're actually working on where appropriate, and ensure that you're working in the appropriate root folder of the library.
 
 ### Updating your `package.json`
 
@@ -103,11 +103,11 @@ This defaults to a C++/WinRT project. If you want to create a C# based native mo
 npx react-native-windows-init --version latest --projectType lib --language cs --overwrite
 ```
 
-That's it, you should be able to open `windows\MyLibrary.sln` and start working on your project.
+That's it, you should be able to open `windows\NativeModuleSample.sln` and start working on your project.
 
 ### Testing your Build
 
-To make sure that everything is working, you'll want to try building `MyLibrary`. First you'll want to make sure you've chosen a supported platform:
+To make sure that everything is working, you'll want to try building `NativeModuleSample`. First you'll want to make sure you've chosen a supported platform:
 
 1. At the top, change the `Solution Platform` to `x86` or `x64`.
 1. In the `Build` menu, select `Build Solution`.
@@ -137,7 +137,7 @@ If you are writing a C++/WinRT module and have added any NuGet package dependenc
 
 #### Option 1: Create a new test app
 1. Follow the [getting started guide](getting-started.md) to create a new React Native Windows app.
-2. Run `npm i <module-local-path> --save` (e.g. `npm i D:\MyLibrary --save`) to install the local module.
+2. Run `npm i <module-local-path> --save` (e.g. `npm i D:\NativeModuleSample --save`) to install the local module.
 3. [Link the native module](native-modules-using.md).
 
 #### Option 2: Adding Windows support to existing sample app

--- a/website/versioned_docs/version-0.60/native-modules-setup.md
+++ b/website/versioned_docs/version-0.60/native-modules-setup.md
@@ -9,7 +9,7 @@ original_id: native-modules-setup
 > - [Native Module Sample in microsoft/react-native-windows-samples](https://github.com/microsoft/react-native-windows-samples/tree/master/samples/NativeModuleSample)
 > - [Sample App in microsoft/react-native-windows/packages/microsoft-reactnative-sampleapps](https://github.com/microsoft/react-native-windows/tree/master/packages/sample-apps)
 
-This guide will help set you up with the Visual Studio infrastructure to author your own stand-alone native module for React Native Windows. In this document we'll be creating the scaffolding for a `MyLibrary` native module.
+This guide will help set you up with the Visual Studio infrastructure to author your own stand-alone native module for React Native Windows. In this document we'll be creating the scaffolding for a `NativeModuleSample` native module.
 
 ## Development Environment
 
@@ -25,7 +25,7 @@ Otherwise, if you're creating a new project from scratch, the quickest way is to
 
 ```cmd
 yarn global add create-react-native-module
-create-react-native-module MyLibrary
+create-react-native-module NativeModuleSample
 ```
 
 Now you'll have a new native module project under `react-native-my-library`. Be sure to look at the command output for further steps you'll want to do before publishing the project.
@@ -53,9 +53,9 @@ If you're planning on writing your native module in C#, you'll want to choose `W
 
 If you're planning on writing your native module in C++, you'll want to choose `Windows Runtime Component (C++/WinRT)`. **Important:** You want **C++/WinRT**, not **C++/CX**. Do not choose `Windows Runtime Component (C++/CX)`. If you don't see the **C++/WinRT** project type, make sure you installed the _C++/WinRT Visual Studio Extension_ linked to above under [Development Environment](#development-environment).
 
-1. Set the `Project Name` to `MyLibrary`.
+1. Set the `Project Name` to `NativeModuleSample`.
 1. Set the `Location` to the native module directory you created earlier.
-1. Set the `Solution Name` to `MyLibrary`.
+1. Set the `Solution Name` to `NativeModuleSample`.
 1. Click `Create`.
 
 Next you'll be prompted to select the versions of Windows you'll support. This should match the values for React Native Windows:
@@ -63,7 +63,7 @@ Next you'll be prompted to select the versions of Windows you'll support. This s
 1. Set the `Target version` to `Windows 10, version 1903 (10.0; Build 18362)`.
 1. Set the `Minimum version` to `Windows 10 Creators Update (10.0; Build 15063)`.
 
-You should now have a new `MyLibrary` solution file at `.\MyLibrary\MyLibrary.sln` and a `MyLibrary` project at `.\MyLibrary\MyLibrary\MyLibrary.csproj` for C# or `.\MyLibrary\MyLibrary\MyLibrary.vcxproj` for C++.
+You should now have a new `NativeModuleSample` solution file at `.\NativeModuleSample\NativeModuleSample.sln` and a `NativeModuleSample` project at `.\NativeModuleSample\NativeModuleSample\NativeModuleSample.csproj` for C# or `.\NativeModuleSample\NativeModuleSample\NativeModuleSample.vcxproj` for C++.
 
 For C++/WinRT project:
 
@@ -75,8 +75,8 @@ For C++/WinRT project:
 Now, we want to rename the root directory of the Windows native code to `windows` to match the peer `android` and `ios` directories:
 
 1. Close the solution with `File` > `Close Solution`.
-1. Rename that top `MyLibrary` directory `windows`.
-1. Re-open the solution file at `windows\MyLibrary.sln`.
+1. Rename that top `NativeModuleSample` directory `windows`.
+1. Re-open the solution file at `windows\NativeModuleSample.sln`.
 
 Now it's time to add React Native Windows into the solution.
 
@@ -85,7 +85,7 @@ Now it's time to add React Native Windows into the solution.
 We're going to add several React Native Windows projects to your solution. So to avoid confusing them with your own code, we're first going to create a solution folder called `ReactNative`:
 
 1. Open the Solution Explorer sidebar.
-1. Right-click on `Solution 'MyLibrary'` at the top.
+1. Right-click on `Solution 'NativeModuleSample'` at the top.
 1. Select `Add` > `New Solution Folder`.
 1. Name the folder `ReactNative`.
 
@@ -115,14 +115,14 @@ For each project, you'll do the following:
 1. Select `Add` > `Existing Project...`.
 1. Select the project file and click `Open`.
 
-You now have all of the React Native Windows projects to your solution. Next we're going to reference them in our `MyLibrary` project.
+You now have all of the React Native Windows projects to your solution. Next we're going to reference them in our `NativeModuleSample` project.
 
 ### Referencing React Native Windows in your Project
 
 The only project reference you **must** add is `Microsoft.ReactNative`. To add the reference:
 
 1. Open the Solution Explorer sidebar.
-1. Right-click on your `MyLibrary` project.
+1. Right-click on your `NativeModuleSample` project.
 1. Select `Add` > `Reference`.
 1. Select `Projects` on the left-hand side.
 1. Check the box next to `Microsoft.ReactNative`.
@@ -131,7 +131,7 @@ The only project reference you **must** add is `Microsoft.ReactNative`. To add t
 After you've added the reference, you need to make sure it doesn't copy itself into your build (otherwise it'll cause build conflicts down the line when you're trying on consume your library):
 
 1. Open the Solution Explorer sidebar.
-1. Under your `MyLibrary` project, expand the `References`.
+1. Under your `NativeModuleSample` project, expand the `References`.
 1. Right-click on `Microsoft.ReactNative`.
 1. Select `Properties`.
 1. Under `Build`, Change `Copy Local` to `False`.
@@ -141,7 +141,7 @@ Now, you're technically ready to go, but in order to improve the developer exper
 If you're writing in C#, you'll want to add `Microsoft.ReactNative.SharedManaged`:
 
 1. Open the Solution Explorer sidebar.
-1. Right-click on your `MyLibrary` project.
+1. Right-click on your `NativeModuleSample` project.
 1. Select `Add` > `Reference`.
 1. Select `Shared Projects` on the left-hand side.
 1. Check the box next to `Microsoft.ReactNative.SharedManaged`.
@@ -150,7 +150,7 @@ If you're writing in C#, you'll want to add `Microsoft.ReactNative.SharedManaged
 If you're writing in C++, you'll want to add `Microsoft.ReactNative.Cxx`:
 
 1. Open the Solution Explorer sidebar.
-1. Right-click on your `MyLibrary` project.
+1. Right-click on your `NativeModuleSample` project.
 1. Select `Add` > `Reference`.
 1. Select `Shared Projects` on the left-hand side.
 1. Check the box next to `Microsoft.ReactNative.Cxx`.

--- a/website/versioned_docs/version-0.61/native-modules-setup.md
+++ b/website/versioned_docs/version-0.61/native-modules-setup.md
@@ -10,7 +10,7 @@ original_id: native-modules-setup
 > - [Native Module Sample in microsoft/react-native-windows-samples](https://github.com/microsoft/react-native-windows-samples/tree/master/samples/NativeModuleSample)
 > - [Sample App in microsoft/react-native-windows/packages/microsoft-reactnative-sampleapps](https://github.com/microsoft/react-native-windows/tree/master/packages/sample-apps)
 
-This guide will help set you up with the Visual Studio infrastructure to author your own stand-alone native module for React Native Windows. In this document we'll be creating the scaffolding for a `MyLibrary` native module.
+This guide will help set you up with the Visual Studio infrastructure to author your own stand-alone native module for React Native Windows. In this document we'll be creating the scaffolding for a `NativeModuleSample` native module.
 
 ## Development Environment
 
@@ -47,18 +47,18 @@ The steps to create a new native module library project are:
 Follow the official React Native instructions at https://reactnative.dev/docs/native-modules-setup.
 
 ```bat
-npx create-react-native-module --module-name "MyLibrary" MyLibrary
-cd MyLibrary
+npx create-react-native-module --module-name "NativeModuleSample" NativeModuleSample
+cd NativeModuleSample
 yarn install
 ```
 
-Now you'll have a new native module project under `MyLibrary`. Be sure to look at the command output for further steps you'll want to do before publishing the project.
+Now you'll have a new native module project under `NativeModuleSample`. Be sure to look at the command output for further steps you'll want to do before publishing the project.
 
 At this point, follow the steps below to add Windows support to the newly created library.
 
 ## Adding Windows support to an existing library
 
-> The steps below are written as if you're working with the `MyLibrary` example above, in the root folder of the project. Substitute the name of the library you're actually working on where appropriate, and ensure that you're working in the appropriate root folder of the library.
+> The steps below are written as if you're working with the `NativeModuleSample` example above, in the root folder of the project. Substitute the name of the library you're actually working on where appropriate, and ensure that you're working in the appropriate root folder of the library.
 
 ### Updating your package.json
 
@@ -122,9 +122,9 @@ If you're planning on writing your native module in C++, you'll want to choose `
 > 
 > If you don't see the `Windows Runtime Component (C++/WinRT)` project type, go back and install the _C++/WinRT Visual Studio Extension_ under [Development Environment](#development-environment).
 
-1. Set the `Project Name` to `MyLibrary`.
+1. Set the `Project Name` to `NativeModuleSample`.
 1. Set the `Location` to the native module directory (typically the same directory with the `ios` and `android` sub-directories).
-1. Set the `Solution Name` to `MyLibrary`.
+1. Set the `Solution Name` to `NativeModuleSample`.
 1. Click `Create`.
 
 Next you'll be prompted to select the versions of Windows you'll support. This should match the values for React Native Windows, which as of version 0.61, are:
@@ -132,7 +132,7 @@ Next you'll be prompted to select the versions of Windows you'll support. This s
 1. Set the `Target version` to `Windows 10, version 1903 (10.0; Build 18362)`.
 1. Set the `Minimum version` to `Windows 10 Creators Update (10.0; Build 15063)`.
 
-You should now have a new `MyLibrary` solution file at `.\MyLibrary\MyLibrary.sln` and a `MyLibrary` project at `.\MyLibrary\MyLibrary\MyLibrary.csproj` for C# or `.\MyLibrary\MyLibrary\MyLibrary.vcxproj` for C++.
+You should now have a new `NativeModuleSample` solution file at `.\NativeModuleSample\NativeModuleSample.sln` and a `NativeModuleSample` project at `.\NativeModuleSample\NativeModuleSample\NativeModuleSample.csproj` for C# or `.\NativeModuleSample\NativeModuleSample\NativeModuleSample.vcxproj` for C++.
 
 Additionally, for C++/WinRT projects, you'll need to change the following:
 
@@ -144,8 +144,8 @@ Additionally, for C++/WinRT projects, you'll need to change the following:
 Now, before we go any further we'll want to rename the root directory of the Windows native code to `windows` to match the peer `android` and `ios` directories:
 
 1. Close the solution with `File` > `Close Solution`.
-1. Rename that top `MyLibrary` directory `windows`.
-1. Re-open the solution file at `windows\MyLibrary.sln`.
+1. Rename that top `NativeModuleSample` directory `windows`.
+1. Re-open the solution file at `windows\NativeModuleSample.sln`.
 
 > *Optional*: Whether you're creating a new native module from scratch, or adding windows support to an existing ios/android module, if you're using git, you'll want to add Visual Studio-specific entries in your project's `.gitignore`.
 > The simplest way to do this is to get [VisualStudio.gitignore](https://raw.githubusercontent.com/github/gitignore/master/VisualStudio.gitignore) and save it as `windows\.gitignore`.
@@ -157,7 +157,7 @@ Now it's time to add React Native Windows into the solution.
 We're going to add several React Native Windows projects to your solution. So to avoid confusing them with your own code, we're first going to create a solution folder called `ReactNative`:
 
 1. Open the Solution Explorer sidebar.
-1. Right-click on `Solution 'MyLibrary'` at the top.
+1. Right-click on `Solution 'NativeModuleSample'` at the top.
 1. Select `Add` > `New Solution Folder`.
 1. Name the folder `ReactNative`.
 
@@ -190,14 +190,14 @@ For each project, you'll do the following:
 When you are done, your solution should look like this:
 ![native module dependencies](assets/native-module-dependencies61.png)
 
-You now have all of the React Native Windows projects to your solution. Next we're going to reference them in our `MyLibrary` project.
+You now have all of the React Native Windows projects to your solution. Next we're going to reference them in our `NativeModuleSample` project.
 
 ### Referencing React Native Windows in your Project
 
 The only project reference you **must** add is `Microsoft.ReactNative`. To add the reference:
 
 1. Open the Solution Explorer sidebar.
-1. Right-click on your `MyLibrary` project.
+1. Right-click on your `NativeModuleSample` project.
 1. Select `Add` > `Reference`.
 1. Select `Projects` on the left-hand side.
 1. Check the box next to `Microsoft.ReactNative`.
@@ -206,7 +206,7 @@ The only project reference you **must** add is `Microsoft.ReactNative`. To add t
 After you've added the reference, you need to make sure it doesn't copy itself into your build (otherwise it'll cause build conflicts down the line when you're trying on consume your library):
 
 1. Open the Solution Explorer sidebar.
-1. Under your `MyLibrary` project, expand the `References`.
+1. Under your `NativeModuleSample` project, expand the `References`.
 1. Right-click on `Microsoft.ReactNative`.
 1. Select `Properties`.
 1. Under `Build`, Change `Copy Local` to `False`.
@@ -216,7 +216,7 @@ Now, you're technically ready to go, but in order to improve the developer exper
 If you're writing in C#, you'll want to add `Microsoft.ReactNative.SharedManaged`:
 
 1. Open the Solution Explorer sidebar.
-1. Right-click on your `MyLibrary` project.
+1. Right-click on your `NativeModuleSample` project.
 1. Select `Add` > `Reference`.
 1. Select `Shared Projects` on the left-hand side.
 1. Check the box next to `Microsoft.ReactNative.SharedManaged`.
@@ -225,7 +225,7 @@ If you're writing in C#, you'll want to add `Microsoft.ReactNative.SharedManaged
 If you're writing in C++, you'll want to add `Microsoft.ReactNative.Cxx`:
 
 1. Open the Solution Explorer sidebar.
-1. Right-click on your `MyLibrary` project.
+1. Right-click on your `NativeModuleSample` project.
 1. Select `Add` > `Reference`.
 1. Select `Shared Projects` on the left-hand side.
 1. Check the box next to `Microsoft.ReactNative.Cxx`.
@@ -233,7 +233,7 @@ If you're writing in C++, you'll want to add `Microsoft.ReactNative.Cxx`:
 
 ### Testing your Build
 
-To make sure that everything is working, you'll want to try building `MyLibrary`. First you'll want to make sure you've chosen a supported platform:
+To make sure that everything is working, you'll want to try building `NativeModuleSample`. First you'll want to make sure you've chosen a supported platform:
 
 1. At the top, change the `Solution Platform` to `x86` or `x64`.
 1. In the `Build` menu, select `Build Solution`.
@@ -252,7 +252,7 @@ When you added a reference to the `Microsoft.ReactNative` project in VS (and to 
 
 We want to instead search for the root directory of `react-native-windows`.
 
-Open your project file (`windows\MyLibrary\MyLibrary.vcxproj` for C++ or `windows\MyLibrary\MyLibrary.csproj` for C#) in a text editor.
+Open your project file (`windows\NativeModuleSample\NativeModuleSample.vcxproj` for C++ or `windows\NativeModuleSample\NativeModuleSample.csproj` for C#) in a text editor.
 
 You're going to insert the following:
 
@@ -327,7 +327,7 @@ If you are writing a C++/WinRT module you'll see references to NuGet packages in
 
 #### Option 1: Create a new test app
 1. Follow the [getting started guide](getting-started.md) to create a new React Native Windows app.
-2. Run `npm i <module-local-path> --save` (e.g. `npm i D:\MyLibrary --save`) to install the local module.
+2. Run `npm i <module-local-path> --save` (e.g. `npm i D:\NativeModuleSample --save`) to install the local module.
 3. [Link the native module](native-modules-using.md).
 
 #### Option 2: Adding Windows support to existing sample app

--- a/website/versioned_docs/version-0.62/native-modules-setup.md
+++ b/website/versioned_docs/version-0.62/native-modules-setup.md
@@ -10,7 +10,7 @@ original_id: native-modules-setup
 > - [Native Module Sample in microsoft/react-native-windows-samples](https://github.com/microsoft/react-native-windows-samples/tree/master/samples/NativeModuleSample)
 > - [Sample App in microsoft/react-native-windows/packages/microsoft-reactnative-sampleapps](https://github.com/microsoft/react-native-windows/tree/master/packages/sample-apps)
 
-This guide will help set you up with the Visual Studio infrastructure to author your own stand-alone native module for React Native Windows. In this document we'll be creating the scaffolding for a `MyLibrary` native module.
+This guide will help set you up with the Visual Studio infrastructure to author your own stand-alone native module for React Native Windows. In this document we'll be creating the scaffolding for a `NativeModuleSample` native module.
 
 ## Development Environment
 
@@ -47,18 +47,18 @@ The steps to create a new native module library project are:
 Follow the official React Native instructions at https://reactnative.dev/docs/native-modules-setup.
 
 ```bat
-npx create-react-native-module --module-name "MyLibrary" MyLibrary
-cd MyLibrary
+npx create-react-native-module --module-name "NativeModuleSample" NativeModuleSample
+cd NativeModuleSample
 yarn install
 ```
 
-Now you'll have a new native module project under `MyLibrary`. Be sure to look at the command output for further steps you'll want to do before publishing the project.
+Now you'll have a new native module project under `NativeModuleSample`. Be sure to look at the command output for further steps you'll want to do before publishing the project.
 
 At this point, follow the steps below to add Windows support to the newly created library.
 
 ## Adding Windows support to an existing library
 
-> The steps below are written as if you're working with the `MyLibrary` example above, in the root folder of the project. Substitute the name of the library you're actually working on where appropriate, and ensure that you're working in the appropriate root folder of the library.
+> The steps below are written as if you're working with the `NativeModuleSample` example above, in the root folder of the project. Substitute the name of the library you're actually working on where appropriate, and ensure that you're working in the appropriate root folder of the library.
 
 ### Updating your package.json
 
@@ -122,9 +122,9 @@ If you're planning on writing your native module in C++, you'll want to choose `
 > 
 > If you don't see the `Windows Runtime Component (C++/WinRT)` project type, go back and install the _C++/WinRT Visual Studio Extension_ under [Development Environment](#development-environment).
 
-1. Set the `Project Name` to `MyLibrary`.
+1. Set the `Project Name` to `NativeModuleSample`.
 1. Set the `Location` to the native module directory (typically the same directory with the `ios` and `android` sub-directories).
-1. Set the `Solution Name` to `MyLibrary`.
+1. Set the `Solution Name` to `NativeModuleSample`.
 1. Click `Create`.
 
 Next you'll be prompted to select the versions of Windows you'll support. This should match the values for React Native Windows, which as of version 0.62, are:
@@ -132,7 +132,7 @@ Next you'll be prompted to select the versions of Windows you'll support. This s
 1. Set the `Target version` to `Windows 10, version 1903 (10.0; Build 18362)`.
 1. Set the `Minimum version` to `Windows 10 Fall Creators Update (10.0; Build 16299)`.
 
-You should now have a new `MyLibrary` solution file at `.\MyLibrary\MyLibrary.sln` and a `MyLibrary` project at `.\MyLibrary\MyLibrary\MyLibrary.csproj` for C# or `.\MyLibrary\MyLibrary\MyLibrary.vcxproj` for C++.
+You should now have a new `NativeModuleSample` solution file at `.\NativeModuleSample\NativeModuleSample.sln` and a `NativeModuleSample` project at `.\NativeModuleSample\NativeModuleSample\NativeModuleSample.csproj` for C# or `.\NativeModuleSample\NativeModuleSample\NativeModuleSample.vcxproj` for C++.
 
 Additionally, for C++/WinRT projects, you'll need to change the following:
 
@@ -144,8 +144,8 @@ Additionally, for C++/WinRT projects, you'll need to change the following:
 Now, before we go any further we'll want to rename the root directory of the Windows native code to `windows` to match the peer `android` and `ios` directories:
 
 1. Close the solution with `File` > `Close Solution`.
-1. Rename that top `MyLibrary` directory `windows`.
-1. Re-open the solution file at `windows\MyLibrary.sln`.
+1. Rename that top `NativeModuleSample` directory `windows`.
+1. Re-open the solution file at `windows\NativeModuleSample.sln`.
 
 > *Optional*: Whether you're creating a new native module from scratch, or adding windows support to an existing ios/android module, if you're using git, you'll want to add Visual Studio-specific entries in your project's `.gitignore`.
 > The simplest way to do this is to get [VisualStudio.gitignore](https://raw.githubusercontent.com/github/gitignore/master/VisualStudio.gitignore) and save it as `windows\.gitignore`.
@@ -157,7 +157,7 @@ Now it's time to add React Native Windows into the solution.
 We're going to add several React Native Windows projects to your solution. So to avoid confusing them with your own code, we're first going to create a solution folder called `ReactNative`:
 
 1. Open the Solution Explorer sidebar.
-1. Right-click on `Solution 'MyLibrary'` at the top.
+1. Right-click on `Solution 'NativeModuleSample'` at the top.
 1. Select `Add` > `New Solution Folder`.
 1. Name the folder `ReactNative`.
 
@@ -192,11 +192,11 @@ For each project, you'll do the following:
 When you are done, your solution should look like this:
 ![native module dependencies](assets/native-module-dependencies62.png)
 
-You now have all of the React Native Windows projects to your solution. Next we're going to reference them in our `MyLibrary` project.
+You now have all of the React Native Windows projects to your solution. Next we're going to reference them in our `NativeModuleSample` project.
 
 ### Referencing React Native Windows in your Project
 
-You'll need to edit your project file manually for this step. Open your project file (`windows\MyLibrary\MyLibrary.vcxproj` for C++ or `windows\MyLibrary\MyLibrary.csproj` for C#) in a text editor.
+You'll need to edit your project file manually for this step. Open your project file (`windows\NativeModuleSample\NativeModuleSample.vcxproj` for C++ or `windows\NativeModuleSample\NativeModuleSample.csproj` for C#) in a text editor.
 
 > Normally, you could manually add references to the correct projects using the Visual Studio UI. However you may be required to update your projects to maintain support for new versions of React Native Windows. The props and target files provided here are to help insulate you from future dependency changes.
 
@@ -269,7 +269,7 @@ here:
 +</Target>
 ```
 
-Save your changes and reload the project in Visual Studio. You should now see `Microsoft.ReactNative` in the Solution Explorer under MyLibrary > References.
+Save your changes and reload the project in Visual Studio. You should now see `Microsoft.ReactNative` in the Solution Explorer under NativeModuleSample > References.
 
 #### C#
 
@@ -327,11 +327,11 @@ here:
 +</Target>
 ```
 
-Save your changes and reload the project in Visual Studio. You should now see `Microsoft.ReactNative` in the Solution Explorer under MyLibrary > References.
+Save your changes and reload the project in Visual Studio. You should now see `Microsoft.ReactNative` in the Solution Explorer under NativeModuleSample > References.
 
 ### Testing your Build
 
-To make sure that everything is working, you'll want to try building `MyLibrary`. First you'll want to make sure you've chosen a supported platform:
+To make sure that everything is working, you'll want to try building `NativeModuleSample`. First you'll want to make sure you've chosen a supported platform:
 
 1. At the top, change the `Solution Platform` to `x86` or `x64`.
 1. In the `Build` menu, select `Build Solution`.
@@ -357,7 +357,7 @@ If you are writing a C++/WinRT module and have added any NuGet package dependenc
 
 #### Option 1: Create a new test app
 1. Follow the [getting started guide](getting-started.md) to create a new React Native Windows app.
-2. Run `npm i <module-local-path> --save` (e.g. `npm i D:\MyLibrary --save`) to install the local module.
+2. Run `npm i <module-local-path> --save` (e.g. `npm i D:\NativeModuleSample --save`) to install the local module.
 3. [Link the native module](native-modules-using.md).
 
 #### Option 2: Adding Windows support to existing sample app

--- a/website/versioned_docs/version-0.63/native-modules-setup.md
+++ b/website/versioned_docs/version-0.63/native-modules-setup.md
@@ -10,7 +10,7 @@ original_id: native-modules-setup
 > - [Native Module Sample in microsoft/react-native-windows-samples](https://github.com/microsoft/react-native-windows-samples/tree/master/samples/NativeModuleSample)
 > - [Sample App in microsoft/react-native-windows/packages/microsoft-reactnative-sampleapps](https://github.com/microsoft/react-native-windows/tree/master/packages/sample-apps)
 
-This guide will help set you up with the Visual Studio infrastructure to author your own stand-alone native module for React Native Windows. In this document we'll be creating the scaffolding for a `MyLibrary` native module.
+This guide will help set you up with the Visual Studio infrastructure to author your own stand-alone native module for React Native Windows. In this document we'll be creating the scaffolding for a `NativeModuleSample` native module.
 
 ## Development Environment
 
@@ -47,18 +47,18 @@ The steps to create a new native module library project are:
 Follow the official React Native instructions at https://reactnative.dev/docs/native-modules-setup.
 
 ```bat
-npx create-react-native-module --module-name "MyLibrary" MyLibrary
-cd MyLibrary
+npx create-react-native-module --module-name "NativeModuleSample" NativeModuleSample
+cd NativeModuleSample
 yarn install
 ```
 
-Now you'll have a new native module project under `MyLibrary`. Be sure to look at the command output for further steps you'll want to do before publishing the project.
+Now you'll have a new native module project under `NativeModuleSample`. Be sure to look at the command output for further steps you'll want to do before publishing the project.
 
 At this point, follow the steps below to add Windows support to the newly created library.
 
 ## Adding Windows support to an existing library
 
-> The steps below are written as if you're working with the `MyLibrary` example above, in the root folder of the project. Substitute the name of the library you're actually working on where appropriate, and ensure that you're working in the appropriate root folder of the library.
+> The steps below are written as if you're working with the `NativeModuleSample` example above, in the root folder of the project. Substitute the name of the library you're actually working on where appropriate, and ensure that you're working in the appropriate root folder of the library.
 
 ### Updating your package.json
 
@@ -122,9 +122,9 @@ If you're planning on writing your native module in C++, you'll want to choose `
 >
 > If you don't see the `Windows Runtime Component (C++/WinRT)` project type, go back and install the _C++/WinRT Visual Studio Extension_ under [Development Environment](#development-environment).
 
-1. Set the `Project Name` to `MyLibrary`.
+1. Set the `Project Name` to `NativeModuleSample`.
 1. Set the `Location` to the native module directory (typically the same directory with the `ios` and `android` sub-directories).
-1. Set the `Solution Name` to `MyLibrary`.
+1. Set the `Solution Name` to `NativeModuleSample`.
 1. Click `Create`.
 
 Next you'll be prompted to select the versions of Windows you'll support. This should match the values for React Native Windows, which as of version 0.62, are:
@@ -132,7 +132,7 @@ Next you'll be prompted to select the versions of Windows you'll support. This s
 1. Set the `Target version` to `Windows 10, version 1903 (10.0; Build 18362)`.
 1. Set the `Minimum version` to `Windows 10 Fall Creators Update (10.0; Build 16299)`.
 
-You should now have a new `MyLibrary` solution file at `.\MyLibrary\MyLibrary.sln` and a `MyLibrary` project at `.\MyLibrary\MyLibrary\MyLibrary.csproj` for C# or `.\MyLibrary\MyLibrary\MyLibrary.vcxproj` for C++.
+You should now have a new `NativeModuleSample` solution file at `.\NativeModuleSample\NativeModuleSample.sln` and a `NativeModuleSample` project at `.\NativeModuleSample\NativeModuleSample\NativeModuleSample.csproj` for C# or `.\NativeModuleSample\NativeModuleSample\NativeModuleSample.vcxproj` for C++.
 
 Additionally, for C++/WinRT projects, you'll need to change the following:
 
@@ -144,8 +144,8 @@ Additionally, for C++/WinRT projects, you'll need to change the following:
 Now, before we go any further we'll want to rename the root directory of the Windows native code to `windows` to match the peer `android` and `ios` directories:
 
 1. Close the solution with `File` > `Close Solution`.
-1. Rename that top `MyLibrary` directory `windows`.
-1. Re-open the solution file at `windows\MyLibrary.sln`.
+1. Rename that top `NativeModuleSample` directory `windows`.
+1. Re-open the solution file at `windows\NativeModuleSample.sln`.
 
 > *Optional*: Whether you're creating a new native module from scratch, or adding windows support to an existing ios/android module, if you're using git, you'll want to add Visual Studio-specific entries in your project's `.gitignore`.
 > The simplest way to do this is to get [VisualStudio.gitignore](https://raw.githubusercontent.com/github/gitignore/master/VisualStudio.gitignore) and save it as `windows\.gitignore`.
@@ -157,7 +157,7 @@ Now it's time to add React Native Windows into the solution.
 We're going to add several React Native Windows projects to your solution. So to avoid confusing them with your own code, we're first going to create a solution folder called `ReactNative`:
 
 1. Open the Solution Explorer sidebar.
-1. Right-click on `Solution 'MyLibrary'` at the top.
+1. Right-click on `Solution 'NativeModuleSample'` at the top.
 1. Select `Add` > `New Solution Folder`.
 1. Name the folder `ReactNative`.
 
@@ -191,11 +191,11 @@ For each project, you'll do the following:
 When you are done, your solution should look like this:
 ![native module dependencies](assets/native-module-dependencies63.png)
 
-You now have all of the React Native Windows projects to your solution. Next we're going to reference them in our `MyLibrary` project.
+You now have all of the React Native Windows projects to your solution. Next we're going to reference them in our `NativeModuleSample` project.
 
 ### Referencing React Native Windows in your Project
 
-You'll need to edit your project file manually for this step. Open your project file (`windows\MyLibrary\MyLibrary.vcxproj` for C++ or `windows\MyLibrary\MyLibrary.csproj` for C#) in a text editor.
+You'll need to edit your project file manually for this step. Open your project file (`windows\NativeModuleSample\NativeModuleSample.vcxproj` for C++ or `windows\NativeModuleSample\NativeModuleSample.csproj` for C#) in a text editor.
 
 > Normally, you could manually add references to the correct projects using the Visual Studio UI. However you may be required to update your projects to maintain support for new versions of React Native Windows. The props and target files provided here are to help insulate you from future dependency changes.
 
@@ -268,7 +268,7 @@ here:
 +</Target>
 ```
 
-Save your changes and reload the project in Visual Studio. You should now see `Microsoft.ReactNative` in the Solution Explorer under MyLibrary > References.
+Save your changes and reload the project in Visual Studio. You should now see `Microsoft.ReactNative` in the Solution Explorer under NativeModuleSample > References.
 
 #### C#
 
@@ -326,11 +326,11 @@ here:
 +</Target>
 ```
 
-Save your changes and reload the project in Visual Studio. You should now see `Microsoft.ReactNative` in the Solution Explorer under MyLibrary > References.
+Save your changes and reload the project in Visual Studio. You should now see `Microsoft.ReactNative` in the Solution Explorer under NativeModuleSample > References.
 
 ### Testing your Build
 
-To make sure that everything is working, you'll want to try building `MyLibrary`. First you'll want to make sure you've chosen a supported platform:
+To make sure that everything is working, you'll want to try building `NativeModuleSample`. First you'll want to make sure you've chosen a supported platform:
 
 1. At the top, change the `Solution Platform` to `x86` or `x64`.
 1. In the `Build` menu, select `Build Solution`.
@@ -360,7 +360,7 @@ If you are writing a C++/WinRT module and have added any NuGet package dependenc
 
 #### Option 1: Create a new test app
 1. Follow the [getting started guide](getting-started.md) to create a new React Native Windows app.
-2. Run `npm i <module-local-path> --save` (e.g. `npm i D:\MyLibrary --save`) to install the local module.
+2. Run `npm i <module-local-path> --save` (e.g. `npm i D:\NativeModuleSample --save`) to install the local module.
 3. [Link the native module](native-modules-using.md).
 
 #### Option 2: Adding Windows support to existing sample app

